srcpkgs/dnsmasq/template

```diff
@@ -31,6 +32,8 @@ do_install() {
 	make PREFIX=/usr BINDIR=/usr/bin DESTDIR=${DESTDIR} install
 
 	vsv dnsmasq
+	vsconf dnsmasq.conf.example dnsmasq.conf
```

Comment:
What is the value of this? We already ship the default config in`/etc`.

Reply:
The file in /etc is meant to be modified, so you don't get to keep the original settings around unless you copied it out. It seems like there's an established pattern of storing the default settings for a package in /usr/share/examples which this `vsconf` directive automates.

Reply:
If the config is modified, the new default file will be placed at `/etc/dnsmasq.conf.new.${version}`.